_posts/2019-01-21-computed-side-effects.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Computed Side Effects"
```

Comment:
We should probably reference Vue.js in the title somehow.

Reply:
agreed